custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/README.md

```diff
@@ -3,8 +3,7 @@
 This is a minimal example showing how to export custom metrics using 
 [prometheus text format](https://prometheus.io/docs/instrumenting/exposition_formats/) to Stackdriver
 in a k8s cluster running on GCE or GKE. This example assumes that you already have
-a [custom metrics setup] in your cluster and use legacy Stackdriver resource
-model (Prometheus to Stackdriver doesn't support new resource model).
+a [custom metrics setup] in your cluster.
 
 ## Prometheus dummy exporter
```

Comment:
I think the orders should be 
```
## Deploy sample application

### Option 1 Prometheus dummy exporter
#### Prometheus to stackdriver sidecar container
### Option 2 Exporting metrics to Google Cloud Managed Service for Prometheus
We just need to reference https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#config-mgd-collection instead of writing our own examples.
## Horizontal Pod Autoscaling object

You may have several sentences here to say how to use HPA to query GMP metrics.
...
```